syft/frameworks/torch/tensors/Polynomial.py

```diff
@@ -0,0 +1,77 @@
+class PolynomialTensor:
+
+    """Tensor type which provides polynomial approximation functionalities using Taylor Series expansion
+       since computing exact functions could impose a overhead on computation
+    """
+
+    def sigmoid(self, x: float) -> float:
```

Comment:
this should not take another value and should just be called on self. Alternatively this could be made a class method and take a tensor instead of a float